### PR TITLE
Update about us

### PR DESCRIPTION
**Pull request by KevinChuangCH**
Update AboutUs to current team members.
Individual portfolio undone.
Photo of Tan Wei Jie missing.

### CODE REVIEW

**Comment by shadow2496** [comment]
👍

**Comment by KevinChuangCH** [comment]
Do not merge first, there are some aspects I need to check with the tutor during the tutorial tomorrow.

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -3,53 +3,36 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
+Media Socializer was developed by the https://github.com/orgs/CS2103JAN2018-F12-B3/people[F12-B3] team. +
 _{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
```
Comment (shadow2496) [inline docs/AboutUs.adoc]: This line has to be deleted.

File: docs/AboutUs.adoc [hunk 2]
```diff
@@ -3,53 +3,36 @@
...
-
-Role: Project Advisor
-
-'''
-
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
-{empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
+=== Seunghwan Choi
+image::shadow2496.jpg[width="150", align="left"]
```
Comment (shadow2496) [inline docs/AboutUs.adoc]: The extension of our image files is PNG, not JPG.

File: docs/AboutUs.adoc [hunk 3]
```diff
@@ -3,53 +3,36 @@
...
-Responsibilities: UI
+Responsibilities: In charge of UI + Code quality + Integration
 
 '''
 
-=== Johnny Doe
-image::yijinl.jpg[width="150", align="left"]
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+=== Chuang Cheng Heng
+image::kevinchuangch.jpg[width="150", align="left"]
```
Comment (shadow2496) [inline docs/AboutUs.adoc]: .jpg -> .png

File: docs/AboutUs.adoc [hunk 4]
```diff
@@ -3,53 +3,36 @@
...
-Role: Developer +
-Responsibilities: Dev Ops + Threading
-
-'''
-
-=== Benson Meier
-image::yl_coder.jpg[width="150", align="left"]
-{empty}[http://github.com/yl-coder[github]] [<<johndoe#, portfolio>>]
+=== Tan Wei Jie
+image::yijinl.jpg[width="150", align="left"]
```
Comment (shadow2496) [inline docs/AboutUs.adoc]: Could you change the name to `nethergale.jpg`?

**Comment by KevinChuangCH** [comment]
Included photo of Tan Wei Jie.

File: docs/team/chuangchengheng.adoc [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+== PROJECT: Media Socializer
+
+---
+
+== Overview
+
+Media Socializer is a desktop address book application used to help social media users to keep track of all the social media platform they are using. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```
Comment (Nethergale) [inline docs/team/chuangchengheng.adoc]: Should pluralize "social media platform" to "social media platforms".

**Comment by KevinChuangCH** [comment]
Picture formats corrected from .jpg to .png

**Pull request merged**